src/App.js

```diff
@@ -1,12 +1,15 @@
 import React, { Component } from "react";
-import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
 import "./App.css";
 import PreHeader from "./components/header/PreHeader";
 import TopNavBar from "./components/header/TopNavBar";
 import Banner from "./components/header/Banner";
 import Footer from "./components/Footer";
 import HomePageComponents from "./pages/home/model/HomePageComponents";
 import ProductPage from "./pages/products/ProductPage";
+import ManPage from "./pages/ManPage/ManPage";
+import WomenPage from "./pages/WomenPage/WomenPage";
+import KidPage from "./pages/KidPage/KidPage";
```

Comment:
### Highly Recommended:
- Change the import variables: `MensPage`, `WomenPage`, `KidsPage`
- Change the File names as well: `./pages/mens/MensPage`, `./pages/womens/WomensPage`, `./pages/kids/KidsPage`